### PR TITLE
add connection keep alive to command method

### PR DESCRIPTION
**Pull request by yujinlim**

### CODE REVIEW

**Comment by pedrobranco** [comment]
AFAIK `bitcoind` does not handle very well multiple `keep-alive` connections (https://github.com/bitcoin/bitcoin/issues/5262).

Despite that, this feature should be optional (a new option via client constructor) and also should be implemented a call with the header to close the connection (care to see [this test](https://github.com/bitcoin/bitcoin/pull/5436/files#diff-58b857d414a55b5d060d30e0185217f9R65)).

Also, please do not commit `dist/*` files since they are committed in each release.

File: dist/src/index.js [hunk 1]
```diff
@@ -166,7 +166,11 @@ class Client {
 
       return this.request.postAsync({
         auth: _lodash2.default.pickBy(this.auth, _lodash2.default.identity),
-        body: body,
+        body: JSON.stringify(body),
```
Comment (pedrobranco) [inline dist/src/index.js]: Do not commit `dist/*` files.

File: src/index.js [hunk 2]
```diff
@@ -123,6 +123,9 @@ class Client {
       return this.request.postAsync({
         auth: _.pickBy(this.auth, _.identity),
         body: JSON.stringify(body),
+        headers: {
```
Comment (pedrobranco) [inline src/index.js]: This must be a client constructor option.

**Comment by yujinlim** [comment]
Thanks for the feedback. Will make the changes and recommit. Thanks

**Comment by yujinlim** [comment]
@pedrobranco i have restructured it, please have a look.

**Comment by ruimarinho** [comment]
@yujinlim I took a deeper look at this issue and we already supports this feature. Just make sure you set the following option when creating the client:

```
new Client({ agentOptions: { keepAlive: true }});
```

The `request` module will pass the options to the http(s).Agent and keep alive will be enabled :)

**Comment by yujinlim** [comment]
ahh, noted.

**Pull request closed without merging**